### PR TITLE
ch1 test

### PR DESCRIPTION
**Pull request by acpurcell**
test
test
test

### CODE REVIEW

File: modules/chapter1/pages/section2.adoc [hunk 1]
```diff
@@ -49,7 +49,7 @@ ansible-galaxy collection install -r requirements.yml
 
 == Setup and Configuration with Automation
 
-You can either use the *_setup.yaml_* playbook to install packages, configure swap, lv, dhcp, http and dns or else run commands manually on the hypervisor.
+You can either use the *_setup.yaml_* playbook to install packages, configure swap, LV, DHCP, HTTP and DNS or else run commands manually on the hypervisor.
```
Comment (acpurcell) [inline modules/chapter1/pages/section2.adoc:60]: Confirm pls.

File: modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -2,8 +2,8 @@
 :navtitle: Home
 
 Welcome to this quick course, _Deploy Mission Critical Application Platform (MCAP)_.
-This course helps you to learn the basic infrastructure required for MCAP and how to deploy nested OpenShift using OpenShift Virtualization.
-After completing this course, you will be able to deploy MCAP solution for production environment.
+This course will help you learn the basic infrastructure required for Red Hat MCAP and how to deploy nested OpenShift using OpenShift Virtualization.
+After completing this course, you will be able to deploy the MCAP solution for production environments.
```
Comment (sarveshpandit) [inline modules/ROOT/pages/index.adoc:6]: Okay

File: modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -25,9 +25,9 @@ Update the Catalog link - ##FIX THIS##
 
 When ordering this catalog item in RHDP:
 
-. Select *Practice/Enablement* for the *Activity* field
-. Select the nearest or any region in the *Region* field
-. Select *n3.xlarge.x86* as the flavor in *Type* field
+. Select *Practice/Enablement* for the *Activity* field.
+. Select the nearest or any region in the *Region* field.
+. Select *n3.xlarge.x86* as the flavor in *Type* field.
```
Comment (sarveshpandit) [inline modules/ROOT/pages/index.adoc:31]: okay

File: modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -41,23 +41,23 @@ For partner support - https://connect.redhat.com/en/support[Help and support,win
 
 For this course, you should have:
 
-* Red Hat Certified Systems Administrator certification or equivalent knowledge of Linux system administration is recommended for all roles.
-* https://rol.redhat.com/rol/app/courses/do280-4.14[Red Hat OpenShift Administration II: Configuring a Production Cluster (DO280),window=read-later] or equivalent knowledge on configuring a production Red Hat OpenShift cluster.
+* Red Hat Certified Systems Administrator certification, or equivalent knowledge of Linux system administration is recommended for all roles.
+* https://rol.redhat.com/rol/app/courses/do280-4.14[Red Hat OpenShift Administration II: Configuring a Production Cluster (DO280),window=read-later], or equivalent knowledge on configuring a production Red Hat OpenShift cluster.
```
Comment (sarveshpandit) [inline modules/ROOT/pages/index.adoc:45]: okay

File: modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -41,23 +41,23 @@ For partner support - https://connect.redhat.com/en/support[Help and support,win
...
 * https://rol.redhat.com/rol/app/technical-overview/do016-4.14[Red Hat OpenShift Virtualization (OCP Virt) Technical Overview (DO016),window=read-later]
-* https://rol.redhat.com/rol/app/courses/do316-4.14[Managing Virtual Machines with Red Hat OpenShift Virtualization (DO316),window=read-later] or equivalent knowledge on how to create virtual machines on Red Hat OpenShift cluster using Red Hat OpenShift Virtualization.
+* https://rol.redhat.com/rol/app/courses/do316-4.14[Managing Virtual Machines with Red Hat OpenShift Virtualization (DO316),window=read-later], or equivalent knowledge on how to create virtual machines on Red Hat OpenShift cluster using Red Hat OpenShift Virtualization.
 * Knowledge on installing Red Hat Advanced Cluster Management (RHACM) operator
```
Comment (sarveshpandit) [inline modules/ROOT/pages/index.adoc:48]: okay

File: modules/ROOT/pages/index.adoc [hunk 6]
```diff
@@ -41,23 +41,23 @@ For partner support - https://connect.redhat.com/en/support[Help and support,win
...
-* Provision multiple tenant clusters on top of the infrastructure cluster
-* Deploy and configure applications and Virtual Machines on Tenant Clusters
-* Test the high availability and resilience of deployed workloads
+* Provision the base infrastructure (HW, Storage, Networking) for the hands-on lab environment.
+* Install utility services needed for the hands-on labs.
+* Configure the base OpenShift infrastructure cluster to host tenant clusters.
+* Configure the Infrastructure Cluster to host Virtual Machines.
+* Provision multiple tenant clusters on top of the infrastructure cluster.
+* Deploy and configure applications and Virtual Machines on Tenant Clusters.
+* Test the high availability and resilience of deployed workloads.
```
Comment (sarveshpandit) [inline modules/ROOT/pages/index.adoc:60]: okay.

File: modules/ROOT/pages/index.adoc [hunk 7]
```diff
@@ -67,4 +67,4 @@ https://www.redhat.com/en/blog/building-mission-critical-application-platform[Bu
 
 https://github.com/arvin-a/openshift-upi-automation.git[Github - openshift-upi-automation,window=read-later]
 
-https://www.redhat.com/en/blog/nested-openshift-using-openshift-virtualization[Nested OpenShift using OpenShift Virtualization,window=read-later]
\ No newline at end of file
+https://www.redhat.com/en/blog/nested-openshift-using-openshift-virtualization[Nested OpenShift using OpenShift Virtualization,window=read-later]
```
Comment (sarveshpandit) [inline modules/ROOT/pages/index.adoc:70]: okay

File: modules/chapter1/pages/index.adoc [hunk 8]
```diff
@@ -1,6 +1,6 @@
 = Hands-on Lab Environment Setup for MCAP
 
-This chapter covers hands-on lab environment setup for MCAP.
+This chapter covers the hands-on lab environment setup for MCAP.
```
Comment (sarveshpandit) [inline modules/chapter1/pages/index.adoc:4]: okay

File: modules/chapter1/pages/section1.adoc
Comment (sarveshpandit) [inline modules/chapter1/pages/section1.adoc:1]: okay

File: modules/chapter1/pages/section2.adoc (hunk 1 above)
Comment (sarveshpandit) [reply modules/chapter1/pages/section2.adoc:60]: Yes.

File: modules/chapter1/pages/section2.adoc
Comment (sarveshpandit) [inline modules/chapter1/pages/section2.adoc:1]: okay

File: modules/chapter1/pages/section3.adoc
Comment (sarveshpandit) [inline modules/chapter1/pages/section3.adoc:1]: Okay

File: modules/chapter1/pages/section4.adoc
Comment (sarveshpandit) [inline modules/chapter1/pages/section4.adoc:1]: okay

File: modules/chapter2/pages/section1.adoc
Comment (sarveshpandit) [inline modules/chapter2/pages/section1.adoc:1]: okay

File: modules/chapter2/pages/section2.adoc
Comment (sarveshpandit) [inline modules/chapter2/pages/section2.adoc:1]: okay

File: modules/chapter2/pages/section3.adoc
Comment (sarveshpandit) [inline modules/chapter2/pages/section3.adoc:1]: okay

File: modules/chapter2/pages/section4.adoc
Comment (sarveshpandit) [inline modules/chapter2/pages/section4.adoc:1]: okay

File: modules/chapter3/pages/section1.adoc
Comment (sarveshpandit) [inline modules/chapter3/pages/section1.adoc:1]: okay

File: modules/chapter3/pages/section2.adoc
Comment (sarveshpandit) [inline modules/chapter3/pages/section2.adoc:1]: Okay

File: modules/chapter3/pages/section3.adoc
Comment (sarveshpandit) [inline modules/chapter3/pages/section3.adoc:1]: okay

File: modules/chapter3/pages/section4.adoc
Comment (sarveshpandit) [inline modules/chapter3/pages/section4.adoc:1]: okay

File: modules/chapter3/pages/section5.adoc
Comment (sarveshpandit) [inline modules/chapter3/pages/section5.adoc:1]: okay

**Review by sarveshpandit — APPROVED** [review]
Approve the changes

**Pull request merged**